docs/install.rst

```diff
@@ -62,13 +72,13 @@ package:
    $ conda activate plenoptic
    $ conda install -c conda-forge jupyterlab ipywidgets torchvision
 
-   This is easy but, if you have multiple conda environments and want to use
-   Jupyter notebooks in each of them, it will take up a lot of space.
+   This is the easiest but, if you have multiple virtual environments and want
+   to use Jupyter notebooks in each of them, it will take up a lot of space.
 
 2. Use `nb_conda_kernels
```

Comment:
running `conda activate` is just to make sure they're in the right environment (if they just opened the terminal, it's not a problem, but if they've been doing something else, it might be)

Reply:
I've tried to add a bit more explanation of the options and what their advantages are, is this any clearer?